test/common/README.md

```diff
@@ -63,9 +65,7 @@ failures.
 Platform normalizes the `dd` command
 
 ### enoughTestMem
-* return [&lt;Boolean>]
```

Comment:
Aren't we losing information here (type of the property)?

Reply:
I'll follow the convention in our docs and restore these lines but remove the word `return`.